src/pages/userInfo.js

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import { TStoDate } from "components/utilities";
+import "css/userInfo.css";
+
+function UserInfo() {
+  const [info, setInfo] = useState({});
+
+  useEffect(() => {
+
+    // A fetch function to get JWT info from server
+    const getInfo = async () => {
+      fetch(window.config.userinfo, {
+        'method': 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'Authorization': undefined
```

Comment:
Use `undefined` for right now. It seems like we still need `Authorization` in the header.